SESHealth/app/src/main/res/layout/activity_setup.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
Probably not a big thing now, but maybe wrap this inside a scrollview. Haven't tested it on a device but just wondering how it will look if the orientation is landscape.